modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -1,15 +1,173 @@
-= Field Level Encryption from the SDK
-:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
+= Field Level Encryption from the Node.js SDK
+:description:  A practical guide for getting started with Field-Level Encryption, showing how to encrypt and decrypt JSON fields using the Node.js SDK.
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+:page-aliases: ROOT:encrypting-using-sdk.adoc
 
 [abstract]
-{description}
+{description}
 
+For a high-level overview of this feature, see our xref:concept-docs:encryption.adoc[Encryption] page.
 
-Client-side implementation of Field Level Encryption is available in the previous version of the SDK.
-It will be enabled in a future release of the third generation SDK.
+[#package]
+== Packaging
+The Node.js SDK works together with the https://github.com/couchbase/node-couchbase-encryption[Node Couchbase Encryption^] library to provide support for encryption and decryption of JSON fields.
+This library includes cryptographic algorithms you can use out of the box, and provides a framework for implementing your own crypto components.
 
-A _Developer Preview_ is available in the xref:3.0@java-sdk:howtos:encrypting-using-sdk.adoc[Java SDK].
+NOTE: The encryption code is packaged as an optional library because it is released under a https://github.com/couchbase/node-couchbase-encryption/blob/master/LICENSE[different license^] from the Couchbase Node SDK.

```

Comment:
Can we check this (I know it's what we've written elsewhere - but isn't it the case that it's because strong crypto has packaging and exporting restrictions?)